app/policies/access_request_policy.rb

```diff
@@ -15,4 +15,5 @@ def create?
 
   alias_method :index?, :approve?
   alias_method :show?, :approve?
+  alias_method :destroy?, :approve?
```

Comment:
test?

Reply:
This is covered in the `approve` spec so I think we're good. Moving on...